sum/front/src/utils/sumCaller.ts

```diff
@@ -0,0 +1,7 @@
+export async function sum(
```

Comment:
maybe you could add a little comment explaining why this function is mocked for now

Reply:
sure
done